cmd/bucket-replication.go

```diff
@@ -851,8 +852,10 @@ func (p *ReplicationPool) queueReplicaTask(oi ObjectInfo) {
 		return
 	}
 	select {
-	case p.replicaCh <- oi:
+	case <-GlobalContext.Done():
+		return
 	default:
+		p.replicaCh <- oi
```

Comment:
This can block @poornas why did you change it like this? and it can hold up a go-routine we can't have that.

Reply:
if server receives Ctrl-C, there is a panic here if replication is in progress. We close the channels right away after global context is cancelled.

Reply:
Yes but it will block if the channel is full @poornas 

We have to keep both in select and default should be to ignore the channel blockage..